flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/catalog/GenericInMemoryCatalogTest.java

```diff
@@ -1104,6 +1104,43 @@ public void testDropFunction_FunctionNotExist_ignored() throws Exception {
 		catalog.dropDatabase(db1, false);
 	}
 
+	// ------ statistics ------
```

Comment:
Well, I think the added test is written differently as others, which gives such an expression. However, it covers all the added implementation methods, as you can see. What's missing, though, is some negative test cases, which I will create a followup JIRA for this.

Reply:
> Well, I think the added test is written differently as others, which gives such an expression. However, it covers all the added implementation methods, as you can see. What's missing, though, is some negative test cases, which I will create a followup JIRA for this.

I think we need to have a consistent style for unit tests in this class, otherwise it's hard for other developers to figure out the two styles and extend/modify the testing code